content/feedId.ttl

```diff
@@ -9,6 +9,6 @@
       Example: `CER/ADA-USD`
 
       The ID can also be extended to a
-      [versioned feed ID](versionedFeedId).
+      [versioned feed ID](#versionedFeedId).
     ''' ;
     skos:related "versionedFeedId".
```

Comment:
This is still not correct. the value is not a string. It is a reference. 
```
skos:related :versionedFeedId
```

Reply:
I have fixed this convention throughout